### PR TITLE
AWS Load Balancer Controller ssl redirect using annotation

### PR DESCRIPTION
**Pull request by pshanoop**
## What is the problem I am trying to address?
Adds support for AWS Load Balancer Controller ssl-redirect using annotation

## How is the fix applied?

Added new value `.Values.ingress.extraPath` where we can specify custome `serviveName` and `servicePort`. This way we can have ssl-redirect mentioned by AWS Load Balancer Controller [documentation](https://kubernetes-sigs.github.io/aws-load-balancer-controller/latest/guide/tasks/ssl_redirect/) 

Another value added  `.Values.ingress.customPath` to customize default path for service, ALB uses glob matching so to match all paths we need default path as `/*`. 

## What GitHub issue(s) does this PR fix or close?

Fixes https://github.com/gomods/athens/issues/1685

### CODE REVIEW

File: charts/athens-proxy/README.md [hunk 1]
```diff
@@ -48,6 +48,33 @@ Available options:
 - [.netrc file support](https://docs.gomods.io/install/install-on-kubernetes/#netrc-file-support)
 - [gitconfig support](https://docs.gomods.io/install/install-on-kubernetes/#gitconfig-support)
 
+### [AWS Load Balancer Controller](https://kubernetes-sigs.github.io/aws-load-balancer-controller/latest/) ssl redirect using annotation
+Using `.Values.ingress.extraPaths` you can difine extra paths to the each hosts. So now we can define
```
Comment (arschles) [inline charts/athens-proxy/README.md:52]: s/difine/define

**Review by arschles — APPROVED** [review]
@pshanoop I'm not experienced with AWS but the helm changes look good to me. Can you fix the conflict?

**Comment by arschles** [comment]
@pshanoop there's one conflict in here, but if you can fix that, this PR is ready to go 😄

**Comment by pshanoop** [comment]
@arschles  I didn't see your first comment. 

Fixed conflict and typo. Thanks for reviewing. 👍🏾

**Comment by manugupt1** [comment]
Hi @pshanoop  Would you be able to update the base-branch and I will be able to merge it after that.

thanks!

**Comment by DrPsychick** [comment]
Hey @pshanoop , is this still an issue? And if so, are you willing to do a new PR on the new Chart repo? https://github.com/gomods/athens-charts

**Pull request closed without merging**